src/_codux/boards/app/app.board.tsx

```diff
@@ -0,0 +1,18 @@
+import { createBoard } from '@wixc3/react-board';
+import { App } from '../../../App';
+import { MemoryRouter } from 'react-router-dom';
+
+export default createBoard({
+   name: 'App',
+   Board: () => (
+      <MemoryRouter>
+         <App />,
+      </MemoryRouter>
```

Comment:
This board has an error

![image](https://github.com/DanielGovrin/ecommerce/assets/67858097/761135a2-6834-43f1-a87b-5f0d69a5630c)